my-profile-middleware-impl/src/main/java/edu/wisc/my/profile/dao/EmergencyContactMiddlewareDaoImpl.java

```diff
@@ -48,7 +48,8 @@ public EmergencyContactMiddlewareDaoImpl(DataSource ds) {
       //return saved content
       return emergencyContacts;
     } else {
-      throw new Exception("There was an issue saving the emergency contacts");
+      throw new Exception("Error saving contacts for netid '" + netId + "'; "
+          + "Contacts that failed to save were " + emergencyContacts + ".");
```

Comment:
The product [already logs successfully saved emergency contact information](https://github.com/UW-Madison-DoIT/my-profile/blob/38d351d48d3388ae1498f8e486ec54f3a1171250/my-profile-webapp/src/main/java/edu/wisc/my/profile/web/EmergencyContactInformationController.java).